decapodes/generate.jl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env -S julia --project=. --startup=no --color yes
+import Pkg; Pkg.instantiate();
+import ACSets.InterTypes: InterTypes, @intertypes, generate_module, PydanticTarget, JSONTarget
+
+JSON_DIR = joinpath((@__DIR__), "json/")
+PYTHON_DIR = joinpath((@__DIR__), "python/")
+
+write(PYTHON_DIR * "/intertypes.py", InterTypes.INTERTYPE_PYTHON_MODULE)
```

Comment:
For platform independence, we should use https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.joinpath